rewrite-gradle/src/test/java/org/openrewrite/gradle/UpgradeDependencyVersionTest.java

```diff
@@ -373,18 +373,6 @@ void upgradesVariablesDefinedInExtraProperties() {
         rewriteRun(
           buildGradle(
             """
-              buildscript {
```

Comment:
I'd remove this part from the new test and potentially change the names of the test methods, so this diff becomes minimal

Reply:
No, I did this on purpose.
- One has an `ext` (that is Extra Properties) in the root of the Gradle file
- The other has the `ext` in the `buildscript`

The first case did already work, the second case did not.